ophyd/epics_motor.py

```diff
@@ -145,8 +145,18 @@ def move(self, position, wait=True, **kwargs):
             If motion fails other than timing out
         '''
         self._started_moving = False
+        status = None
 
+        def dmov_callback(old_value, value, **kwargs):
+            self.log.debug(f'DEBUG: dmov_callback(): new={value} old={old_value}')
+            if self._started_moving and value == 1:
+                self.motor_done_move.clear_sub(dmov_callback)
+                status._finished(success=True)
+                status.done = True
```

Comment:
Calling `_finished` sets this. It need not and probably should not be done explicitly. We should make this a property so that that is unambigous.

Reply:
Agreed.  It's gone.

Reply:
Done here: https://github.com/bluesky/ophyd/pull/784